index.html

```diff
@@ -4818,230 +4817,149 @@ <h3>Base Context</h3>
-      "@id": "https://w3id.org/security#EcdsaSecp256k1Signature2019",
-      "@context": {
-        "@version": 1.1,
-        "@protected": true,
-
-        "id": "@id",
-        "type": "@type",
-
-        "sec": "https://w3id.org/security#",
-        "xsd": "http://www.w3.org/2001/XMLSchema#",
-
-        "challenge": "sec:challenge",
-        "created": {"@id": "http://purl.org/dc/terms/created", "@type": "xsd:dateTime"},
-        "domain": "sec:domain",
-        "expires": {"@id": "sec:expiration", "@type": "xsd:dateTime"},
-        "jws": "sec:jws",
-        "nonce": "sec:nonce",
-        "proofPurpose": {
-          "@id": "sec:proofPurpose",
-          "@type": "@vocab",
-          "@context": {
-            "@version": 1.1,
-            "@protected": true,
-
-            "id": "@id",
-            "type": "@type",
-
-            "sec": "https://w3id.org/security#",
-
-            "assertionMethod": {"@id": "sec:assertionMethod", "@type": "@id", "@container": "@set"},
-            "authentication": {"@id": "sec:authenticationMethod", "@type": "@id", "@container": "@set"}
-          }
+        "holder": {
+          "@id": "https://www.w3.org/2018/credentials#holder",
+          "@type": "@id"
         },
-        "proofValue": "sec:proofValue",
-        "verificationMethod": {"@id": "sec:verificationMethod", "@type": "@id"}
-      }
-    },
-
-    "EcdsaSecp256r1Signature2019": {
-      "@id": "https://w3id.org/security#EcdsaSecp256r1Signature2019",
-      "@context": {
-        "@version": 1.1,
-        "@protected": true,
-
-        "id": "@id",
-        "type": "@type",
-
-        "sec": "https://w3id.org/security#",
-        "xsd": "http://www.w3.org/2001/XMLSchema#",
-
-        "challenge": "sec:challenge",
-        "created": {"@id": "http://purl.org/dc/terms/created", "@type": "xsd:dateTime"},
-        "domain": "sec:domain",
-        "expires": {"@id": "sec:expiration", "@type": "xsd:dateTime"},
-        "jws": "sec:jws",
-        "nonce": "sec:nonce",
-        "proofPurpose": {
-          "@id": "sec:proofPurpose",
-          "@type": "@vocab",
-          "@context": {
-            "@version": 1.1,
-            "@protected": true,
-
-            "id": "@id",
-            "type": "@type",
-
-            "sec": "https://w3id.org/security#",
-
-            "assertionMethod": {"@id": "sec:assertionMethod", "@type": "@id", "@container": "@set"},
-            "authentication": {"@id": "sec:authenticationMethod", "@type": "@id", "@container": "@set"}
-          }
+        "proof": {
+          "@id": "https://w3id.org/security#proof",
+          "@type": "@id",
+          "@container": "@graph"
         },
-        "proofValue": "sec:proofValue",
-        "verificationMethod": {"@id": "sec:verificationMethod", "@type": "@id"}
+        "verifiableCredential": {
+          "@id": "https://www.w3.org/2018/credentials#verifiableCredential",
+          "@type": "@id",
+          "@container": "@graph"
+        }
```

Comment:
I'll fix this in PR #1004.

Reply:
Hmm, I'm looking at PR #1004 now... I'm not sure we want `termsOfUse` there? I'd expect it to kick in w/ a `VerifiableCredential` type scoped context instead of via the term. What am I missing? Are you saying we should have both trigger the behaviour?